base/file.jl

```diff
@@ -518,14 +518,15 @@ is an open file object for this path.
 mktemp(parent)
 
 """
-    mktempdir(parent=tempdir(); prefix=$(repr(temp_prefix)))
+    mktempdir(parent=tempdir(); prefix=$(repr(temp_prefix)), remove_on_exit = false)
```

Comment:
```suggestion
    mktempdir(parent=tempdir(); prefix=$(repr(temp_prefix)), remove_on_exit=false)
```
and elsewhere, won't comment on each